### ISSUE TITLE
Can't install LSP from package control

### ISSUE THREAD

**Issue by chendonming** [sublimelsp/LSP#1415]
The url `https://bitbucket.org/teddy_beer_maniac/sublime-text-dependency-jinja2/get/2.8.0.zip` is 404
now I how to install it? thanks

**Comment by chendonming** [comment]
```
Package Control: Error downloading package. HTTP error 404 downloading https://bitbucket.org/teddy_beer_maniac/sublime-text-dependency-jinja2/get/2.8.0.zip.
error: Package Control
```

**Comment by rwols** [comment]
The dependency is

https://github.com/wbond/package_control_channel/blob/7487b83bc3de0c9bfeda82caba12fc6d4c58816b/repository/dependencies.json#L809-L821

which seems to be online just fine https://github.com/packagecontrol/jinja2

I don't know why Package Control is trying to download that file.

**Comment by rchl** [comment]
Since you seem to be from China, maybe the reason for that is that you've tried to work around problems related to the great firewall and added some custom repository.

That dependency is now on github so it looks like you have some outdated cached repository or custom repository added.

Here is the same issue which should have been resolved a year ago: https://forum.sublimetext.com/t/python-jinja2-package-install-problem/29120/7

Maybe try running the `Package Control: Remove Repository` to see if you have any custom ones added. Fresh install shouldn't have any.

**Comment by rwols** [comment]
@chendonming were you able to solve the problem?

**Comment by rwols** [comment]
@chendonming I will assume you managed to fix your issue. Feel free to open a new issue or join the Discord channel if you're still blocked on this.
